binderhub/utils.py

```diff
@@ -136,6 +139,35 @@ def url_path_join(*pieces):
     return result
 
 
+def maybe_future(obj):
```

Comment:
BinderHub depends on JupyterHub so could we import this utility from there instead of copying?

Reply:
That's a good idea. I try it now. Actually we can do the same for `Callable` traitlet.